micro-ui/web/micro-ui-internals/packages/modules/workbench/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 
 
+## [2.0.0-rc19-02] [02-Sept-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)